src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -1603,8 +1592,9 @@ def loc():
         -----
         To meet the nature of lazy evaluation:
 
-        - Snowpark pandas ``.loc`` ignores out-of-bounds indexing for all types of indexers (while pandas ``.loc``
+        - Snowpark pandas ``.loc`` ignores out-of-bounds indexing for row indexers (while pandas ``.loc``
           may raise KeyError). If all values are out-of-bound, an empty result will be returned.
+        - Out-of-bounds indexing for columns will still raise a KeyError the same way pandas does.
```

Comment:
I can't add a comment on the actual lines but can you correct "absert" --> "absent" on lines 1601 and 1602? Thanks!

Reply:
Done!